packages/core/package.json

```diff
@@ -18,6 +18,7 @@
     "react-dom": "^15.0.1 || ^0.14"
   },
   "devDependencies": {
+    "@blueprintjs/core": "^1.0.1",
```

Comment:
whoa this is really bad... instead we should set up local resolution in webpack.